src/library_fs.js

```diff
@@ -104,13 +104,9 @@ FS.staticInit();` +
     // This is set to false during `preInit`, allowing you to modify the
     // filesystem freely up until that point (e.g. during `preRun`).
     ignorePermissions: true,
+    #if FS_DEBUG
```

Comment:
For consistency these should be indented all the way at the bottom,
```suggestion
#if FS_DEBUG
```
However, I do agree it looks better this way :smile: Perhaps we should open an issue to discuss the indentation? For this PR, meanwhile, let's do it the old way for now.